objects/oPlayer/Create_0.gml

```diff
@@ -1,3 +1,10 @@
 walkSpeed = 8;
 image_speed = 0;
-image_index = 3;
\ No newline at end of file
+image_index = 3;
+
+max_cooldown_1 = 120;
+cooldown_1 = 0;
+
+max_cooldown_2 = 16;
+cooldown_2 = 0;
```

Comment:
These variable names could probably be something like aoe_max_cooldown, aoe_cooldown, fireball_max_cooldown and fireball_cooldown?